### ISSUE TITLE
Error: 'softmax' not yet implemented or not found in the registry. Did you forget to import the kernel?

### ISSUE THREAD

**Issue by Buddhilive** [tensorflow/tfjs#2910]
I get this error only if I implement _@tensorflow/tfjs-node_ . But if I'm using only _@tensorflow/tfjs_ library to train my model, the error doesn't show up and the model trains without any issue.

` UnhandledPromiseRejectionWarning: Error: 'softmax' not yet implemented or not found in the registry. Did you forget to import the kernel?`

**Comment by rthadur** [comment]
Can you please provide these details : 

1. Model if possible
2. Reproduction code.
3. System configuration.

**Comment by Buddhilive** [comment]
I figured out the issue. I was trying to import both the packages as follows:

`const tf = require('@tensorflow/tfjs');
require('@tensorflow/tfjs-node');`

and I changed it as follows:

`const tf = require('@tensorflow/tfjs-node');`

completely removing _@tensorflow/tfjs_ from my project. Then it start working.
